rskj-core/src/test/java/co/rsk/peg/PegUtilsTest.java

```diff
@@ -637,4 +631,328 @@ void test_getTransactionType_flyover_segwit() {
         // Assert
         Assertions.assertEquals(PegTxType.UNKNOWN, pegTxType);
     }
+
+    @Test
```

Comment:
Let's move all these tests to a new class `PegUtilsAllUTXOsToFedAreAboveMinimumPeginValueTest`. That way we avoid ending up with a huge test class

Reply:
Updated!